### ISSUE TITLE
Problem with GeoJson associated to DataSet on TerraLib

### ISSUE THREAD

**Issue by avancinirodrigo** [TerraME/terrame#1403]
Currently is not possible create cellular space and to use attribute fill from a geojson data in TerraView. The problem might be associated to DataSet which its name does not work as a shapefile, although both are handled by OGR.
Another fact, the OGR is handled by GDAL in the TerraLib. It can be observed in Ogr::DataSource::open():

``` cpp
m_ogrDS = (GDALDataset*)GDALOpenEx(path.c_str(), GDAL_OF_UPDATE, NULL, NULL, NULL);
```

Associated to #1217.

**Comment by avancinirodrigo** [comment]
This was resolved in TerraLib 5.3

**Comment by pedro-andrade-inpe** [comment]
Good. Please close it together with #2179.
